CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
+  - Remove start_date attribute
```

Comment:
Does a real deprecated attribute raise a warning and is still managed or does it raise a fatal error ?